### ISSUE TITLE
Make it embeddable

### ISSUE THREAD

**Issue by peq42** [FoldingAtHome/fah-client-bastet#283]
It'd be interesting if FaH was embedable.

Imagine if devs could make FaH run in the background of their game/application, on a low priority/less cores/limited GPU usage, bringing a bunch of processing power to the cause while not consuming much CPU/GPU on your average user's machine

**Comment by jcoffland** [comment]
The client code could be restructured such that there was a C/C++ API for running the client.  However, this would only be justified is someone actually planned to use it on a platform with a large number of users.  It's kind of a chicken-or-the-egg problem but we have to be careful how we allocate our limited development resources.

**Comment by kbernhagen** [comment]
Wouldn't this create another fah malware kerfuffle?

**Comment by arisu3** [comment]
> Wouldn't this create another fah malware kerfuffle?

I'm not aware of this. Did someone put FAH in malware?

**Comment by muziqaz** [comment]
No, FAH might be put in by game Dev into the game, and that game Dev does not tell anyone that they done it. Community finds out, all hell breaks loose, fah gets bad name for that, as if we are responsible
